### PR TITLE
Feat8 add toast

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Added Toast

### CODE REVIEW

**Review by anjali1102 — APPROVED** [review]

File: src/Components/Toast.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+export default function ({ message, type }) {
+  console.log("Comes Here");
+  return toast(message, {
+    position: toast.POSITION.TOP_CENTER,
+    autoClose: 1000,
+    type: type,
+    transition: Slide,
+    closeOnClick: true,
+    pauseOnHover: false,
```
Comment (MaruthiPothuganti) [inline src/Components/Toast.js:12]: Toast can be directly invoked by an inbuilt function called **toast**. Check this [LINK](https://www.knowledgehut.com/blog/web-development/react-toastify)

**Review by dev-enforced — APPROVED** [review]
The code looks good and well structured !! Good work Shruti on the toast feature. Do check the deploy preview whether it is working or not

File: src/context/like-context.js [hunk 2]
```diff
@@ -21,11 +22,18 @@ const LikedProvider = ({ children }) => {
         );
         if (response.status === 201) {
           setLikedList(response.data.likes);
+          Toast({ type: "success", message: "Added to Liked Videos" });
         }
       } catch (error) {
         console.log("error", error);
-        navigate("/login");
+        Toast({
```
Comment (MaruthiPothuganti) [inline src/context/like-context.js:29]: So we don't need to pass the config of type. We can directly do this.
image

**Review by MaruthiPothuganti — APPROVED** [review]
Toast feature Works perfect. PR looks good to merge.

File: src/Components/Toast.js (hunk 1 above)
Comment (ShrutiLakhani) [reply src/Components/Toast.js:12]: Thank you for sharing it :) I will work with it in next project

**Pull request merged**